pkgs/top-level/all-packages.nix

```diff
@@ -17911,6 +17911,10 @@ in
 
     cpupower = callPackage ../os-specific/linux/cpupower { };
 
+    cpupower-gui = python3Packages.callPackage ../os-specific/linux/cpupower/gui.nix {
+      inherit (pkgs) meson;
+    };
+
```

Comment:
does the application need to be exported as a kernel module?
```
$ nix-env -f /home/jon/.cache/nixpkgs-review/pr-100120-4/nixpkgs -qaP --xml --out-path --show-trace --meta
18 packages added:
linuxPackages_latest_xen_dom0.cpupower-gui (init at 0.9.1) linuxPackages_4_4.cpupower-gui (init at 0.9.1) linuxPackages_testing_bcachefs.cpupower-gui (init at 0.9.1) linuxPackages_xen_dom0.cpupower-gui (init at 0.9.1) linuxPackages_4_14.cpupower-gui (init at 0.9.1) linuxPackages_5_8.cpupower-gui (init at 0.9.1) linuxPackages_4_9.cpupower-gui (init at 0.9.1) linuxPackages_zen.cpupower-gui (init at 0.9.1) linuxPackages.cpupower-gui (init at 0.9.1) linuxPackages_hardkernel_4_14.cpupower-gui (init at 0.9.1) linuxPackages_xen_dom0_hardened.cpupower-gui (init at 0.9.1) linuxPackages-libre.cpupower-gui (init at 0.9.1) linuxPackages_latest_xen_dom0_hardened.cpupower-gui (init at 0.9.1) linuxPackages_hardened.cpupower-gui (init at 0.9.1) linuxPackages_latest-libre.cpupower-gui (init at 0.9.1) linuxPackages_5_9.cpupower-gui (init at 0.9.1) linuxPackages_4_19.cpupower-gui (init at 0.9.1) linuxPackages_latest_hardened.cpupower-gui (init at 0.9.1)
```

I think this could live outside modules as just a gui application.

Reply:
That's correct. It doesn't depend on a specific kernel either.

Reply:
Sorry, I left it next to cpupower as related. I've now moved it out of the `linuxPackages` section and next to other os-specific applications.